packages/translator/src/Charcoal/Translator/Translator.php

```diff
@@ -242,15 +262,32 @@ public function translateChoice($val, $number, array $parameters = [], $domain =
             $locale = $this->getLocale();
         }
 
+        // Convert old parameters
+        $oldParameters = $parameters;
+        $parameters = [];
+        foreach ($oldParameters as $key => $value) {
+            if ($key != '%count%') {
+                $key = preg_replace('/^%(\w+)%$/', '$1', $key);
+                $parameters[$key] = $value;
+            } else {
+                $parameters[$key] = $value;
+            }
+        }
```

Comment:
This can be simplified:


```suggestion
        foreach ($oldParameters as $key => $value) {
            if ($key !== '%count%') {
                $key = preg_replace('/^%(\w+)%$/', '$1', $key);
            }

            $parameters[$key] = $value;
        }
```

I don't understand why here we have an exclusion for `%count%` but not in `translationChoice()`.